kong/plugins/httplog/handler.lua

```diff
@@ -0,0 +1,15 @@
+local BasePlugin = require "kong.plugins.base_plugin"
+local log = require "kong.plugins.httplog.log"
+
+local HttpLogHandler = BasePlugin:extend()
+
+function HttpLogHandler:new()
+  HttpLogHandler.super.new(self, "httplog")
+end
+
+function HttpLogHandler:log(conf)
+  HttpLogHandler.super.log(self)
+  log.execute(conf)
```

Comment:
It's really unnecessary to nest this in `log.lua` when your actual function is like 5 lines long. Just keep it simple and put the code in the `handler:log`